### PR TITLE
handle case with no dataset

### PR DESCRIPTION
**Pull request by jburel**
Handle the case when the dataset is not set.

See https://www.openmicroscopy.org/qa2/qa/feedback/29385/

### CODE REVIEW

File: src/main/java/org/openmicroscopy/shoola/env/data/model/ImportableFile.java [hunk 1]
```diff
@@ -152,7 +153,10 @@ public FileObject getOriginalFile()
 	 */
 	public boolean isDatasetCreated()
 	{
-		return dataset.getId() >= 0;
+		if (dataset == null || dataset.getId() < 0) {
+			return false;
+		}
+		return  true;
```
Comment (dominikl) [inline src/main/java/org/openmicroscopy/shoola/env/data/model/ImportableFile.java:159]: Could be simplified to `return dataset != null && dataset.getId() > -1`.

**Comment by dominikl** [comment]
Thanks JM. The if-else isn't strictly necessary, but looks fine.

File: src/main/java/org/openmicroscopy/shoola/env/data/model/ImportableFile.java (hunk 1 above)
Comment (jburel) [reply src/main/java/org/openmicroscopy/shoola/env/data/model/ImportableFile.java:159]: true

**Comment by dominikl** [comment]
👍 Good to merge.

**Pull request merged**